earth_enterprise/src/server/wsgi/serve/publish/publish_manager_helper.py

```diff
@@ -617,14 +617,13 @@ def _QueryTargetDbDetailsByPath(self, target_path):
 
     target_details = {}
     query_string = ("""SELECT db_table.host_name, db_table.db_name,
-              virtual_host_table.virtual_host_name, target_table.serve_wms, publish_context_table.ec_default_db
-              FROM target_table, target_db_table, db_table, virtual_host_table, publish_context_table
+              virtual_host_table.virtual_host_name, target_table.serve_wms
```

Comment:
Just change the select clause to not bring back that column.  The from clause and the where clause can stay the same.